ci/README.md

```diff
@@ -0,0 +1,40 @@
+# CLI commands for CI process
+
+## Usage
+
+### Gather diagnostics
+
+Gather DC/OS services diagnostic from specific cluster by running  following command:
+
+```bash
+$ ci_cli.py --type=diagnostic <service> [<service> ...]
+```
+
+- *service* is DC/OS service name, e.g., `cassandra`, `elastic` and `hdfs`
+
+
+### Install data services (`install_data_services.sh`)
+
+Bulk installation required for DC/OS services in parallel.
+
+```bash
+$ install_data_services.sh <service> [<service> ...]
+```
+- *service* is DC/OS service name, e.g., _cassandra_, _elastic_ and _hdfs_
```

Comment:
You're using underscores here and backticks in the preceding above.

Reply:
It is markup where underscores mean  `italic` just to highlight